tests/e2e/network/ensureloadbalancer.go

```diff
@@ -35,7 +35,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = FDescribe("Ensure LoadBalancer", func() {
+var _ = Describe("Ensure LoadBalancer", func() {
```

Comment:
`FDescribe` should be used since we want them to be run by default.

same as below